### PR TITLE
style correction: services adjustment, hero adjustment, footer adjust…

### PR DESCRIPTION
**Pull request by vladyslavdobychin**
…ment

### CODE REVIEW

File: dist/css/mobile.css [hunk 1]
```diff
@@ -6,18 +6,17 @@ html {
   scroll-behavior: smooth; }
 
 body {
-  font-family: 'Roboto', sans-serif;
+  font-family: "Roboto", sans-serif;
   background: #fff;
```
Comment (Dysco76) [inline dist/css/mobile.css:10]: Comment

Comment (Dysco76) [reply dist/css/mobile.css:10]: Comment 2

File: scss/main.scss [hunk 2]
```diff
@@ -1,752 +1,8 @@
...
-// }
\ No newline at end of file
+@import "variables";
+@import "utilities";
+@import "about";
+@import "services";
+@import "pages";
+@import "contact";
+@import "footer";
+@import "header_hero";
```
Comment (dmytrosc) [inline scss/main.scss:8]: no new line at end of file

File: scss/_variables.scss [hunk 3]
```diff
@@ -1,7 +1,7 @@
...
-$text-primary: #211F54;
-$text-secondary: #6E7191;
-$blue-color: #0051B2;
-$white-color: #CCCCCC;
+$primary-blue: #0075ff;
+$text-primary: #211f54;
+$text-secondary: #6e7191;
+$blue-color: #0051b2;
+$white-color: #cccccc;
 $header-height: 118px;
```
Comment (dmytrosc) [inline scss/_variables.scss]: no new line at end of file

File: scss/_utilities.scss [hunk 4]
```diff
@@ -1,107 +1,96 @@
...
+  color: $text-primary;
+  font-weight: 700;
 }
 
 .font-heading {
-    font-size: 48px;
-    font-weight: 700;
+  font-size: 48px;
+  font-weight: 700;
 }
```
Comment (dmytrosc) [inline scss/_utilities.scss:95]: no new line at end of file

File: scss/_utilities.scss [hunk 5]
```diff
@@ -1,107 +1,96 @@
...
 body {
-    font-family: 'Roboto', sans-serif;
-    background: $primary-white;
-    line-height: 1.6;
-    box-sizing: border-box;
-    font-size: 18px;
-    font-weight: 400;
+  font-family: "Roboto", sans-serif;
+  background: $primary-white;
+  line-height: 1.6;
```
Comment (dmytrosc) [inline scss/_utilities.scss]: why empty line?

File: scss/_services.scss [hunk 6]
```diff
@@ -1,346 +1,353 @@
...
+
+    li::before {
+      content: "";
+      position: absolute;
+      background-image: url(../img/List.png);
+      width: 26px;
+      height: 26px;
+      left: 0;
+      // transform: translateY(-50%);
+      // top: 50%;
```
Comment (dmytrosc) [inline scss/_services.scss]: what are comments for?

File: scss/_services.scss [hunk 7]
```diff
@@ -1,346 +1,353 @@
...
+      margin-right: 58px;
+    }
 
+    span {
+      color: $primary-blue;
     }
-}
\ No newline at end of file
+  }
+}
```
Comment (dmytrosc) [inline scss/_services.scss:329]: no new line at end of file

File: scss/_pages.scss [hunk 8]
```diff
@@ -1,86 +1,85 @@
...
-}
\ No newline at end of file
+    &__decoration {
+      background-color: $primary-blue;
+      height: 50px;
+      width: 50px;
+      justify-content: flex-end;
+    }
+  }
+}
```
Comment (dmytrosc) [inline scss/_pages.scss:85]: no new line at end of file

File: scss/_footer.scss [hunk 9]
```diff
@@ -1,79 +1,89 @@
...
-        text-decoration: underline;
-        color:#0075FF;
-    }
-}
\ No newline at end of file
+  &__a {
+    text-decoration: underline;
+    color: #0075ff;
+  }
+}
```
Comment (dmytrosc) [inline scss/_footer.scss:89]: no new line at end of file

File: scss/_about.scss [hunk 10]
```diff
@@ -1,54 +1,48 @@
...
-    margin-bottom: 240px;
-    display: flex;
-    justify-content: center;
-    gap: 28px;
-}
\ No newline at end of file
+  display: flex;
+  justify-content: center;
+  gap: 28px;
+}
```
Comment (dmytrosc) [inline scss/_about.scss:45]: no new line at end of file

File: dist/index.html [hunk 11]
```diff
@@ -65,7 +65,10 @@ <h1 class="hero-content__heading">
 
             <div class="phone">
               <p class="action__p">Call us now</p>
-              <h2 class="action__h font-2">(414) 567-2109</h2>
+              <!-- <h2 class="action__h font-2">(414) 567-2109</h2> -->
```
Comment (dmytrosc) [inline dist/index.html]: remove comment

File: scss/main.scss (hunk 2 above)
Comment (vladyslavdobychin) [reply scss/main.scss:8]: Line added

File: scss/_variables.scss (hunk 3 above)
Comment (vladyslavdobychin) [reply scss/_variables.scss]: Line added

File: scss/_utilities.scss (hunk 4 above)
Comment (vladyslavdobychin) [reply scss/_utilities.scss:95]: Line added

File: scss/_services.scss (hunk 7 above)
Comment (vladyslavdobychin) [reply scss/_services.scss:329]: Line added

File: scss/_pages.scss (hunk 8 above)
Comment (vladyslavdobychin) [reply scss/_pages.scss:85]: Line added

File: scss/_footer.scss (hunk 9 above)
Comment (vladyslavdobychin) [reply scss/_footer.scss:89]: Line added

File: scss/_about.scss (hunk 10 above)
Comment (vladyslavdobychin) [reply scss/_about.scss:45]: Line added

File: dist/index.html (hunk 11 above)
Comment (vladyslavdobychin) [reply dist/index.html]: Comment removed

File: scss/_services.scss (hunk 6 above)
Comment (vladyslavdobychin) [reply scss/_services.scss]: Comments removed

File: scss/_header_hero.scss [hunk 12]
```diff
@@ -1,104 +1,105 @@
...
+
+  &__p {
+    color: $text-secondary;
+    text-transform: uppercase;
+  }
+
+  &__h {
+    color: $text-primary;
+  }
+}
```
Comment (dmytrosc) [inline scss/_header_hero.scss:114]: no new line at end of file

File: scss/_services.scss [hunk 13]
```diff
@@ -1,346 +1,349 @@
...
+      width: 26px;
+      height: 26px;
+      left: 0;
+    }
+  }
 
-        }
+  .list2 {
+    color: $text-primary;
+    list-style: none;
```
Comment (dmytrosc) [inline scss/_services.scss]: child li tag already has this list-style: none

File: scss/_services.scss [hunk 14]
```diff
@@ -1,346 +1,349 @@
...
 .list {
-    display: flex;
-    gap: 100px;
-    margin-left: 25px;
-    margin-bottom: 25px;
+  display: flex;
+  gap: 56px;
+  margin-bottom: 48px;
+
+  .list1 {
```
Comment (dmytrosc) [inline scss/_services.scss]: zero difference between .list1 & .list2 - try refactoring into a single class

File: scss/_services.scss (hunk 13 above)
Comment (vladyslavdobychin) [reply scss/_services.scss]: Corrected

File: scss/_services.scss (hunk 14 above)
Comment (vladyslavdobychin) [reply scss/_services.scss]: both columns are not under one class

**Review by dmytrosc — APPROVED** [review]
good job!

**Pull request merged**